examples/intro_example.py

```diff
@@ -0,0 +1,33 @@
+"""
+This example matches the one in the README, and is run
+on travis to ensure that it works on every build.
```

Comment:
why not just convert this into a sphinx gallery example that starts with plot_ so that we don't need to add it to Travis?